src/pages/components/accordion/snippets/usage.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default `
+<Accordion>
+  <AccordionItem title="Panel A">Panel A</AccordionItem>
+  <AccordionItem title="Panel B">Panel B</AccordionItem>
+  <AccordionItem title="Panel C">Panel C</AccordionItem>
+</Accordion>
+`;
```

Comment:
In my component demo PRs design had suggested having the text be something more meaningful to the accordion example, which I kind of liked as well. 
<img width="972" alt="Screen Shot 2020-04-30 at 1 59 50 PM" src="https://user-images.githubusercontent.com/32556167/80748949-5e2a4e80-8aeb-11ea-9e86-9b2537a609de.png">

Reply:
Agreed!